### DISCUSSION TITLE
"Received invalid response from the server."error

### DISCUSSION THREAD

**Discussion by laptiy** [dmitrijkir/mattermost-plugin-calendar#25] [General]
Hi, was trying to install hour plugin, but got "Received invalid response from the server."error, any ideas?

**Comment by dmitrijkir** [comment]
Hi, most likely this is due to the file size. You need to increase the upload file size on nginx. This parameter is MaxFileSize.
https://forum.mattermost.com/t/error-installing-the-mattermost-ai-plugin-post-message-has-size-restrictions/17323
